### DISCUSSION TITLE
Did v1.0.5 change the way the results are being printed?

### DISCUSSION THREAD

**Discussion by neferin12** [cvc5/cvc5#9572] [Q&A] [answered]
I hope this is the place to ask this. I forked and used the node-smtlib library by stanford oval which basically is a node library to create smt-lib code and getting the result for that from cvc5 (cvc4 in the original library). The interfacing is done by creating a child process that runs cvc5, putting the smt-lib code to the stdin of that process and receiving the results from cvc5 by reading and parsing its stdout.  That worked mostly fine, up until version 1.0.5, which seems to have changed something regarding its output, because the library simply doesn't receive any output any more. Do you have any idea what the reason for that might be?

(I know this is probably a bit of a hacky way to do this, its more a proof of concept)

**Comment by ajreynol** [comment, marked as answer]
Can you try `--no-flex-parser --no-stdin-input-per-line` and let me know if that fixes the issue?

We are in the process of updating our parser, which may be the reason for the change in behavior.

**Reply by neferin12** [reply]
Yes, that fixed it, thank you very much
